src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -261,6 +261,38 @@ internal static CompletionResult GetCommandNameCompletionResult(string name, obj
             return new CompletionResult(name, listItem, CompletionResultType.Command, syntax);
         }
 
+        private static bool isNested(PSModuleInfo rootModule, PSModuleInfo childModule)
+        {
+            var nestedModules = rootModule.NestedModules;
```

Comment:
This thinking has been the cause of many performance problems.

This property is accessed twice and it is not a trivial property, therefore it is reasonable and encouraged to use local.  The property implementation is pretty cheap, but getting it twice would result in extra unnecessary code being executed.  And if the property access became more expensive in the future, you wouldn't need to worry about this code.